src/main/java/com/conveyal/gtfs/graphql/GraphQLGtfsSchema.java

```diff
@@ -421,6 +422,7 @@
             .field(MapFetcher.field("entity_id"))
             .field(MapFetcher.field("entity_sequence", GraphQLInt))
             .field(MapFetcher.field("bad_value"))
+            .field(ErrorPriorityFetcher.field("error_type_priority"))
```

Comment:
It seems a little bit overkill to define a custom Fetcher for the `priority` string. It looks like every other string field is being fetched with the `MapFetcher`. Can you describe your thinking here? It seems like unneeded complexity and like we're just introducing a new pattern.